### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by finwo**
Updating dependency and browserify script, with the intention of first getting the repository in a functional state again before fixing the range notation (gt, gte, lt, lte instead of start, end).

- Replaced [level-browserify](https://npmjs.com/package/level-browserify) by [level](https://npmjs.com/package/level) as the project indicated, because installing level-browserify wasn't possible anymore on a recent system (api changes according to node-gyp).
- Fixed invalid stream closure in `test/triple_unicode_store_spec` and `test/triple_store_spec` (replaced listening to `close` event by `end` event).
- Non-major updates on [level-sublevel](https://npmjs.com/package/level-sublevel), [multilevel](https://npmjs.com/package/multilevel), [osenv](https://npmjs.com/package/osenv), [setimmediate](https://npmjs.com/package/setimmediate) & [tmp](https://npmjs.com/package/tmp)

### CODE REVIEW

**Comment by mcollina** [comment]
Are you using this module? Are you interested in maintaining this module going further?
I'm not actively maintaining this so I might consider giving you access.

(There is probably some work to do on CI, test are not being run).

**Comment by finwo** [comment]
I've used it in the past, and interested using it again.

Based on previous usage: This lib is still pretty much the only one that I can simply throw batch of triples into and it just works. Hexadb was inspired by this lib, but the queries are either broken (or I'm missing something). GunDB works, but has a different api style which doesn't readily support searching. I've tried others, but most are tied to RDF these days, trying to handle some magic edge-cases.

While converting to neo4j, jena or mongodb with lots of indexes would probably work, having a matcher on bare triples on top of a key-value store & then converting them freely between triples and objects gives a lot of freedom in storage.

TL;DR; I used to used it, looking to use it again & interested in maintaining for the bugs I come across (and if I have time left, perhaps even some open issues).

**Comment by mcollina** [comment]
I have added you to the team! You can land this yourself. Open an issue/discussion when you would like to cut a new release and we'll organize that as well.

**Comment by finwo** [comment]
Awesome!

This specific PR should not generate a release in my opinion, based on the notion of up-coming changes.

**Comment by finwo** [comment]
Merging, not bumping the version number.
